website/docs/reference/glossary.md

```diff
@@ -0,0 +1,159 @@
+
+A group of operations used to reduce or summarize large data sets.
+See [list of supported aggregation operations and commands here](./supported_commands.md#aggregation-pipelines).
+
+#### aggregation pipeline
+
+A set of operators that lets you perform complex operations that aggregate and summarize values.
+See [list of supported aggregation pipeline operators](./supported_commands.md#aggregation-pipeline-operators) here.
+
+---
+
+### B
+
+#### Beacon
+
+The telemetry service of FerretDB.
+
+#### BSON
+
+Stands for “binary JSON”, which is a serialized binary file format for storing JSON-like documents.
+
+#### BSON types
+
+The list of types that the BSON format supports.
+BSON offers support for additional data types compared to JSON, such as `timestamp`, `date`, `ObjectId`, and `binary`.
+
+---
+
+### C
+
+#### collection
+
+A group of documents in a non-relational database.
+It is comparable to a table in a relational database.
+
+#### CRUD
+
+The four basic operations of a database: Create, Read, Update, and Delete.
+See [Basic FerretDB CRUD operations here](../basic_operations/index.md).
+
+---
+
+### D
+
+#### dance
+
+The integration testing tool of FerretDB, which is named after the Ferret war dance.
+See [dance repository](https://github.com/FerretDB/dance) for more details.
+
+#### database
+
+An organized repository for collections containing its own sets of documents, and data.
+
+#### database command
+
+The set of commands in FerretDB.
+For more information, see [supported commands](./supported_commands.md) for more details.
+
+#### document
+
+A record in a collection that comprises key-value pairs.
+See [Documents](../understanding_ferretdb.md#documents) for more.
+
+#### dot notation
+
+Dot notation is used to reference or access the elements in an array or in an embedded document.
+See [dot notation](../understanding_ferretdb.md#dot-notation) for more details.
+
+---
+
+### F
+
+#### field
+
+Similar to columns in a relational database.
+They are represented as field-value pairs and describe the kind of data in a document.
+
+---
+
+### G
+
+#### github-actions
+
+A repository that holds the shared GitHub Actions across all FerretDB repositories.
+See [github-actions repository](https://github.com/FerretDB/github-actions) for more details.
+
+---
+
+### I
+
+#### index
+
+A data structure used for identifying and querying records in a database.
+
+---
+
+### J
+
+#### JSON
+
+An acronym for JavaScript Object Notation.
+It is a structured data format with human-readable text  to store data objects composed of attribute-value pairs.
+
+#### JSONB
+
+JSONB is a data type that stores JSON data as a decomposed binary format.
+
+#### ObjectId
+
+A defining 12-byte type that ensures singularity and uniques within a collection and are used to represent the default values for the `_id` fields.
+It contains a 4-byte timestamp value measured in seconds, uniquely random machine and process-generated ID, and an incremental counter.
```

Comment:
That is how we generate ObjectId. However ObjectId if specified by a user can be anything except a few exception one of which is nil. Let's double check if anything else is exception.

Reply:
Not even "we", but the driver they use. It might be better just to remove that sentence

Reply:
> ObjectId if specified by a user can be anything
That's not quite true. ObjectId can be only 12-byte type described in this paragraph. `_id` field can have any type except regex and array.

Reply:
Anything that is 12-bytes long :)